proprietary/tokens/src/common/amsterdam/link-type.tokens.json

```diff
@@ -0,0 +1,47 @@
+{
+  "amsterdam": {
+    "link-type": {
+      "color": { "value": "{amsterdam.color.primary-blue}" },
+      "hover": {
+        "color": { "value": "{amsterdam.color.dark-blue}" }
+      },
+      "on-background-dark": {
+        "color": { "value": "{amsterdam.color.primary-white}" },
+        "hover": {
+          "color": { "value": "{amsterdam.color.primary-white}" }
+        }
+      },
+      "on-background-light": {
+        "color": { "value": "{amsterdam.color.primary-black}" },
+        "hover": {
+          "color": { "value": "{amsterdam.color.primary-black}" }
+        }
+      },
+      "regular": {
+        "text-decoration-line": { "value": "underline" },
+        "text-decoration-thickness": { "value": "2px" },
+        "text-underline-offset": { "value": "6px" },
+        "hover": {
+          "text-decoration-thickness": { "value": "3px" },
+          "text-underline-offset": { "value": "5px" }
+        }
+      },
+      "apparent": {
```

Comment:
I understand the quest for a better term than ‘obvious’. That one didn’t stick with me either. We should make time for it now.

The problem with both of these is that they go like ‘that this is a link is so obvious / apparent that it doesn't need an initial underline’. But the visual effect of this link type is the opposite: making it look a bit _less_ obvious than a link. And I think the characteristic of the link visualisation is what we should describe, not the context it’s in.

I think I would propose ‘muted’. It’s a term I come across regularly in design systems – albeit with colours, but we can use it there as well to prevent having two similar terms. It’s short, pretty clear, easy to pronounce, not too particular.

Plenty of [alternatives](https://www.thesaurus.com/browse/muted) though: soft, subtle, plain, dimmed, light, slight, discreet, subdued, silent, quiet, mellow, toned-down.

What would you say?